src/access/profiling/experiment.py

```diff
@@ -52,6 +58,33 @@ class ProfilingExperimentStatus(Enum):
     NEW = 1  # Experiment has been created but not started
     RUNNING = 2  # Experiment is running or is queued
     DONE = 3  # Experiment has finished
+    ARCHIVED = 4  # Experiment has been archived
+
+
+def experiment_directory_walker(path: Path, arcname: Path):
```

Comment:
```suggestion
def experiment_directory_walker(path: Path, archivename: Path):
```

```suggestion
def experiment_directory_walker(path: Path, rxivname: Path):
```

May be a clearer variable name would be better